std/format.d

```diff
@@ -7764,7 +7764,7 @@ if (is(T == float) || is(T == double) || (is(T == real) && T.mant_dig == double.
         int count = exp / 60 + 1;
```

Comment:
I assume that the value of count cannot ever be greater than 18?

Reply:
Indeed. For float it cannot be greater than 3 and for double it cannot be greater than 18. (exp is maximum 127/1023).